adcircpy/forcing/waves/base.py

```diff
@@ -0,0 +1,13 @@
+from os import PathLike
+
+from adcircpy.forcing.base import Forcing
```

Comment:
should be relative import

Reply:
I'm actually favoring absolutes now, let's discuss this.